dist/msi/components/QtClient.wxs

```diff
@@ -85,40 +85,64 @@
     </DirectoryRef>
     <DirectoryRef Id="INSTALLDIR" FileSource="$(var.QtDir)\bin">
 
-      <Component Id="dll.qt5.core">
-        <File DiskId="1" KeyPath="yes" Name="Qt5Core.dll" />
+      <Component Id="dll.qt.core">
+        <File DiskId="1" KeyPath="yes" Name="Qt$(var.QtMajorVer)Core.dll" />
       </Component>
-      <Component Id="dll.qt5.dbus">
-        <File DiskId="1" KeyPath="yes" Name="Qt5DBus.dll" />
+      <Component Id="dll.qt.dbus">
+        <File DiskId="1" KeyPath="yes" Name="Qt$(var.QtMajorVer)DBus.dll" />
       </Component>
-      <Component Id="dll.qt5.gui">
-        <File DiskId="1" KeyPath="yes" Name="Qt5Gui.dll" />
+      <Component Id="dll.qt.gui">
+        <File DiskId="1" KeyPath="yes" Name="Qt$(var.QtMajorVer)Gui.dll" />
       </Component>
-      <Component Id="dll.qt5.network">
-        <File DiskId="1" KeyPath="yes" Name="Qt5Network.dll" />
+      <Component Id="dll.qt.network">
+        <File DiskId="1" KeyPath="yes" Name="Qt$(var.QtMajorVer)Network.dll" />
       </Component>
-      <Component Id="dll.qt5.widgets">
-        <File DiskId="1" KeyPath="yes" Name="Qt5Widgets.dll" />
+      <Component Id="dll.qt.widgets">
+        <File DiskId="1" KeyPath="yes" Name="Qt$(var.QtMajorVer)Widgets.dll" />
       </Component>
-      <Component Id="dll.qt5.winextras">
-        <File DiskId="1" KeyPath="yes" Name="Qt5WinExtras.dll" />
+      <?if $(var.QtMajorVer) = 5 ?>
+      <Component Id="dll.qt.winextras">
+        <File DiskId="1" KeyPath="yes" Name="Qt$(var.QtMajorVer)WinExtras.dll" />
       </Component>
+      <?endif ?>
 
     </DirectoryRef>
     <DirectoryRef Id="INSTALLDIR" FileSource="$(var.QtDir)\plugins">
 
+      <?if $(var.QtMajorVer) = 6 ?>
+      <Directory Id="QTIMAGEFORMATSDIR" Name="imageformats">
+        <Component Id="dll.qt.plugins.imageformats.gif">
+          <File DiskId="1" KeyPath="yes" Name="qgif.dll" />
+        </Component>
+        <Component Id="dll.qt.plugins.imageformats.ico">
+          <File DiskId="1" KeyPath="yes" Name="qico.dll" />
+        </Component>
+        <Component Id="dll.qt.plugins.imageformats.jpeg">
+          <File DiskId="1" KeyPath="yes" Name="qjpeg.dll" />
+        </Component>
+      </Directory>
+      <?endif ?>
```

Comment:
I've got no context for this block; just taking your word for it :smile_cat:

Reply:
No sure why but I didn't bundle those plugins before. Since they aren't part of Qt builds that are used on CI now, I'm not including them for Qt 5.